pkg/apis/core/types.go

```diff
@@ -511,6 +511,8 @@ const (
 	ReadOnlyMany PersistentVolumeAccessMode = "ReadOnlyMany"
 	// can be mounted in read/write mode to many hosts
 	ReadWriteMany PersistentVolumeAccessMode = "ReadWriteMany"
+	// can be mounted read/write mode to exactly 1 pod
```

Comment:
With the new restrictions, IMO a comment here should mention that this mode cannot be used with any other mode.

Reply:
Done, added `// cannot be used in combination with other access modes`